opentech/apply/projects/models.py

```diff
@@ -26,6 +31,10 @@ def create_from_submission(cls, submission):
         project = Project.objects.create(
             submission=submission,
             name=submission.title,
+            contact_email=submission.user.email,
+            contact_legal_name=submission.user.full_name,
+            contact_address=submission.form_data.get('address', ''),
+            value=submission.form_data.get('value', ''),
```

Comment:
so... there is some magic here: https://github.com/OpenTechFund/opentech.fund/blob/767de79cc46550939055576e7c554054e1d3eabc/opentech/apply/funds/models/submissions.py#L336-L358
Meaning we can do `submission.address` and `submission.value`. 

It is essentially doing what you are doing, but hiding the underlying implementation
https://github.com/OpenTechFund/opentech.fund/blob/767de79cc46550939055576e7c554054e1d3eabc/opentech/apply/funds/models/submissions.py#L719-L720

Reply:
When those fields aren't in the `form_data` column we get a `KeyError`, so rather than wrap them in try/excepts I accessed the field directly.  Is there a better way to handle that error path?

Reply:
They are required on the submission so you can to rely on them here if you want to. I would rather it failed a bit nosily than silently in the previous use case, but It doesn't make much difference in this instance I don't think. (it would have already complained trying to progress the application)

I just wanted to highlight the implementation for you incase you missed it.